recipes/pcapplusplus/meta.yaml

```
@@ -0,0 +1,44 @@
+{% set name = "PcapPlusPlus" %}
+{% set version = "22.05" %}
+{% set sha256 = "5f299c4503bf5d3c29f82b8d876a19be7dea29c2aadcb52f2f3b394846c21da9" %}
+
+package:
+  name: {{ name|lower }}
```

We prefer to have native (C++) library packages to have a `lib` prefix.
```suggestion
  name: lib{{ name|lower }}
```